src/klipse/compiler.cljs

```diff
@@ -29,10 +29,11 @@
                     ))
 
 (def repl-opts-noop (merge (replumb/options :browser
-                                             ["/dbg/js" "/js/compiled/out"]
+                                             ["/dbg/js"]
                                              io/no-op)
                             {:warning-as-error false
                              :context :statement
+                             :cache {:src-paths-lookup? true}
```

Comment:
done.
Please merge

On Sun, Apr 10, 2016 at 9:53 AM, raphaelboukara <notifications@github.com>
wrote:

> In src/klipse/compiler.cljs
> <https://github.com/viebel/klipse/pull/21#discussion_r59128396>:
>
> >                                               io/no-op)
> >                              {:warning-as-error false
> >                               :context :statement
> > +                             :cache {:src-paths-lookup? true}
>
> @viebel <https://github.com/viebel> can you please add a comment and/or a
> link to doc?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/viebel/klipse/pull/21/files/00c29cab7f6bc6affed34320f66b75c83fc4e121#r59128396>
>